src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1368,6 +1380,17 @@ class ApplicationManagerImpl
   sync_primitives::Lock audio_pass_thru_lock_;
   sync_primitives::Lock tts_global_properties_app_list_lock_;
   hmi_apis::Common_DriverDistractionState::eType driver_distraction_state_;
+
+  /**
+    * @brief lockScreenDismissalEnabled parameter is not mandatory in Mobile
+    * API.
+    * If message from HMI does not contain lockScreenDismissalEnabled parameter,
+    * lock_screen_dismissal_enabled_ sets by value (kNotExists), otherwise it
+    * sets value which contains in the message.
+    * By deffault this parameter is initialised as
+    * LockScreenDismissalEnabled::kNotExists.
+    */
+  LockScreenDismissalEnabled lock_screen_dismissal_enabled_;
```

Comment:
@IGapchuk Why do we need so long and complicated description just for one variable? Looks that this description for huge class with dozens of methods etc.

Reply:
@IGapchuk make it short with just a purpose of variable

Reply:
@AByzhynar Fixed in 0bb0e987107bdb5669927ed6977a81157f0fbad8.

Reply:
@IGapchuk It is not about how tha value is set...No.. Just describe what do we need this variable for! Everything else can be seen from the code

Reply:
@AByzhynar Fixed in cd078272812e56dbc954d3d9319ac822f24b3093.